include/uapi/sound/sof/abi.h

```diff
@@ -60,5 +60,6 @@
 
 /* SOF ABI magic number "SOF\0". */
 #define SOF_ABI_MAGIC		0x00464F53
+#define SOF_IPC4_ABI_MAGIC	0x34464F53
```

Comment:
maybe you should add comment just like SOF_ABI_MAGIC, `/* SOF IPC4 ABI magic number "SOF4". */`.

Reply:
WIll add it.